packages/oas-utils/src/spec-getters/getExampleFromSchema.test.ts

```diff
@@ -1071,4 +1071,13 @@ describe('getExampleFromSchema', () => {
       },
     })
   })
+
+  it('handles objects in examples and returns them as an array', () => {
+    expect(
+      getExampleFromSchema({
+        type: 'array',
+        examples: [{ 0: 'example 1', 1: 'example 2' }],
```

Comment:
That syntax seems strange and is not what’s in the PR description, pls enlighten me :D

Reply:
seems like using

```yaml
example:
  - example 1
  - example 2
```

is turned into a schema `examples` object, can sync about this if need!

Reply:
oops updated